controllers/web.js

```diff
@@ -32,7 +29,10 @@ module.exports = () => {
 
           //if user need to login
           if (resp.result.action === "connection" || resp.result.action === "welcome") {
-            return Bluebird.resolve(new Button("web_url", `${config.server.url}${config.server.basePath}/authorize?state=${senderId}-web`,"Il faut d'abord se connecter !"));
+            let but = [
```

Comment:
you have a point but this button is temporary, it's for testing purposes and is going to be removed during the next iteration :)